deployments/examples/spark30/cmd/run.sh

```diff
@@ -16,12 +16,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# replace these env var values accordingly
-SPARK_HOME=/Users/example/repository/spark
+SCRIPT_PATH=$(cd `dirname $0`; pwd)
+# set up root directory
+WORK_SPACE_ROOT=`dirname $SCRIPT_PATH`
```

Comment:
This forces a run from within the checked out tree. Which means you remove the flexibility from the user to have or use multiple inputs and setups.
The user should have the flexibility to run to run this from wherever they want on the system.

Reply:
We can give a parameter to let the user specify the location.

Reply:
happy with that too